### ISSUE TITLE
Shortcode "[toc]" Not Working

### ISSUE THREAD

**Issue by jwbats** [zedzedzed/table-of-contents-plus#122]
Hi Michael,

I really want to use your ToC plugin on my website. Unfortunately, only the [sitemap] shortcode works. The [toc] shortcode does not.

I decided to also try it out on a much simpler website, which has much fewer plugins installed, in order to minimize the chances of plugin incompatibility.

Unfortunately, it simply refuses to work. I'm not getting any error output.

I appreciate any help you can give me. What's going wrong?

Sincerely,

Jay

**Comment by jwbats** [comment]
I just managed to get output from your plugin. I ruled out incompatibility with other plugins by doing it on a clean WP install.

The plugin wasn't generating anything because I start my wordpress posts with an H2 header, since the post title itself is already an H1 header.

However, I've seen pages where your plugin generates a ToC just fine, with the starting header being a H3, followed by a H2 with nested H3s.

As a matter of fact, the ToC works fine with a starting H2 tag ("Description") on your own site:

http://dublue.com/plugins/toc/#Scenarios_FAQs

How come that works for others, but not for me?
